google/cloud/aiplatform/docker_utils/run.py

```diff
@@ -110,6 +113,18 @@ def run_prediction_container(
         host_port (int):
             Optional. The port on the host that the port, AIP_HTTP_PORT, inside the container
             will be exposed as. If it's unset, a random host port will be assigned.
+        gpu_count (int):
+            Optional. Number or devices to request. Set to -1 to request all available devices.
+            To use GPU, set either `gpu_count` or `gpu_device_ids`.
+        gpu_device_ids (List[str]):
+            Optional. List of strings for device IDs.
+            To use GPU, set either `gpu_count` or `gpu_device_ids`.
+        gpu_capabilities (List[List[str]]):
+            Optional. List of lists of strings to request capabilities. To use GPU, you need
+            to set this. The global list acts like an OR, and the sub-lists are AND. The driver
+            will try to satisfy one of the sub-lists.
+            Available capabilities for the NVIDIA driver can be found in
+            https://github.com/NVIDIA/nvidia-container-runtime.
```

Comment:
I assume  `gpu_device_ids` and `gpu_capabilities`  correspond to `NVIDIA_VISIBLE_DEVICES` and `NVIDIA_DRIVER_CAPABILITIES` documented here. If it's true, explicitly note this.